source/02.03_datatypes.sub

```diff
@@ -3,19 +3,21 @@ Data Types
 
 The following data types are used in the LAS format definition. Note that these
 data types are conformant to the 1999 ANSI C Language Specification
-(ANSI/ISO/IEC 9899:1999 ("C99")).
-
-* char (1 byte)
-* unsigned char (1 byte)
-* short (2 bytes)
-* unsigned short (2 bytes)
-* long (4 bytes)
-* unsigned long (4 bytes)
-* long long (8 bytes)
-* unsigned long long (8 bytes)
-* float (4 byte IEEE floating point format)
-* double (8 byte IEEE floating point format)
-* string (a variable series of 1 byte characters, ASCII encoded, null-terminated)
+(ANSI/ISO/IEC 9899:1999 ("C99")), as defined in the standard header
+``<stdint.h>``.
+
+* int8_t (1 byte)
```

Comment:
There are some places below where `char` is still used in the context of a string. This seems fine to me, but I'm just noting in case this wasn't intentional.)

Reply:
Yes, the use of `char` in strings was already noted in the datatypes section. I've added a note that `char` should be `int8_t` or `uint8_t` in b677ddf.